docs/modules/ROOT/partials/Intro_to_Mill_Header.adoc

```diff
@@ -1,26 +1,26 @@
+{mill-github-url}[Mill] is a {language} build tool that aims for performance,
```

Comment:
I don't think we should claim it a "Java build tool" in one place and a "Scala build tool" in the other. I don't have a better wording yet, but we should aim for one.

Maybe just "build tool" or "generic build tool". It's running on the JVM, so naturally, it's "(primarily) targetting JVM languages".